includes/blocks/course-theme/class-quiz-pagination.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * File containing the Quiz_Pagination class.
+ *
+ * @package sensei
+ * @since 4.0.0
+ */
+
+namespace Sensei\Blocks\Course_Theme;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use \Sensei_Blocks;
+use Sensei_Quiz;
+
+/**
+ * Class Quiz_Pagination is responsible for rendering the quiz pagination block.
+ */
+class Quiz_Pagination {
+	/**
+	 * Quiz_Pagination constructor.
+	 */
+	public function __construct() {
+		Sensei_Blocks::register_sensei_block(
+			'sensei-lms/quiz-pagination',
+			[
+				'render_callback' => [ $this, 'render' ],
+			]
+		);
+	}
+
+	/**
+	 * Renders the block.
+	 *
+	 * @access private
+	 *
+	 * @return string The block HTML.
+	 */
+	public function render() : string {
+		ob_start();
+		Sensei_Quiz::the_quiz_pagination();
+		$pagination = ob_get_clean();
+		return ( "
+			<form method='POST' enctype='multipart/form-data'>
```

Comment:
Since we're adding it to the `single.php` template, probably we need some check to not write this form on the lesson page, right?

Reply:
Not really. The quiz pagination works as a submit buttons inside a form. Each page button is a form value. If we remove form then quiz pagination will stop working. We ignore the form values if the quiz is already submitted. https://github.com/Automattic/sensei/blob/9ae94fa0fda3e6512b2fb09e58e800831fb6931f/includes/class-sensei-quiz.php#L461

Reply:
I mean that we're rendering the form on the lesson page, without having the quiz pagination.

<img width="1085" alt="Screen Shot 2022-01-11 at 18 36 47" src="https://user-images.githubusercontent.com/876340/149025467-15be9038-7264-403a-bcd8-50b127470569.png">

Reply:
Ohh, of course! 🤦 Fixed. 43de66e2c956a5ef56656f54c8921b1e47faae28